corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -666,3 +668,22 @@ def get_case_types_for_domain(domain, include_deprecated=False):
         deprecated_case_types = get_data_dict_deprecated_case_types(domain)
         all_case_types -= deprecated_case_types
     return all_case_types
+
+
+def get_all_case_types_for_domain(domain):
+    """
+    Returns case types no matter if there is an existing case or not, and any defined in the data dictionary,
+    which includes those referenced in an app and those added manually.
+    Excludes system case types.
+    """
```

Comment:
Can you clarify that this function is meant to pull from the app configuration as the source of truth?

Reply:
13ae7b64894a3d6d0a6ca66f40a98d08adaeb6a5